libs/traits/src/rewards.rs

```diff
@@ -121,14 +121,14 @@ pub trait AccountRewards<AccountId> {
 	/// Type used as balance for all currencies and reward.
 	type Balance;
 
-	/// Type used to identify the currency
-	type CurrencyId;
```

Comment:
No intention to be nitpicking but given that this code is within the context of `rewards.rs`, I don't see the need to prefix this with type with `Reward`

Reply:
But, to be fair, I also noticed that we do the same in other places, with `Balance` for example. So I can revert this naming change and then probably add a `CurrencyOf<T>` instead of `(T::DomainId, T::CurrencyId)`?

Reply:
Gotcha! That makes sense, let's keep it then 👍 thanks

Reply:
I come here a little bit late to clarify the `CurrencyId` naming. From the rewards traits perspective, what you put in the `currency_id` is a `CurrencyId`. Nevertheless, the `pallet-rewards` who implements this, add some domain as a prefix or namespace to that `CurrencyId`. This is why in the pallet the `currency_id` is `(T::DomainId, T::CurrencyId)`.

Reply:
It makes more sense if viewed from the perspective of the rewards trait. I reverted that naming change.